File: Front-end/SIRCADE/src/app/users/interfaces/responses/user-info.response.ts
export interface UserInfoResponse {
  id: number;
  nsa: string;
  grade: string;
  paternalLastName: string;
  maternalLastName: string;
  names: string;
  unityId: string;
  birthDate: Date;
  phone: string;
  email: string;
  cellPhone: string;
  situation: number;
  documentNumber: number;
  maritalStatus: number;
  address: string;
  observation?: string;
  roleId: string;
}

File: Back-end/SIRCADE.ApiCore.Controllers/SIRCADE.ApiCore.Controllers/Users/Responses/UserInfoResponse.cs
﻿using SIRCADE.ApiCore.Models.Users.Enums;

namespace SIRCADE.ApiCore.Controllers.Users.Responses;

public record UserInfoResponse(
    int Id,
    string Nsa,
    int RoleId,
    string Grade,
    string PaternalLastName,
    string MaternalLastName,
    string Names,
    int UnityId,
    DateTime BirthDate,
    string Phone,
    string Email,
    string CellPhone,
    Situation Situation,
    string DocumentNumber,
    MaritalStatus MaritalStatus,
    string? Address,
    string? Observation);